docs/Installation.md

```diff
@@ -5,6 +5,9 @@ The instructions provided below specify the steps for SLES 11 SP4/12/12 SP1/12 S
 _**NOTE:**_
 * make sure you are logged in as user with sudo permissions
 
+_**RECOMMENDED:**_
+Create and activate a virtual environment before instaling python packages. If you are facing permission issues while creating environment inside software-discovery-tool folder, refer Step 6 of Installation.
```

Comment:
And you can remove the reference to step 6 here as well.

Reply:
Ok will do